documentation/book/con-jvm-options.adoc

```diff
@@ -0,0 +1,17 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='con-jvm-options-{context}']
+
+= JVM Options
+
+Use the `jvmOptions` property to configure supported JVM options for the VM on which the component is running.
+
+Supported JVM options help to optimize performance for different platforms and architectures.
+
+For more information on the supported options, see xref:ref-jvm-options-deployment-configuration-kafka[JVM configuration]
```

Comment:
is this reference right? should be it related to kafka mirror maker?

Reply:
Yes. This one is fine.